env.md

```diff
@@ -0,0 +1,118 @@
+# Inventory of Agoric SDK Environment Variables
```

Comment:
"Inventory of" adds little. I suggest striking it.

If each section had a pointer to more info, we could call it an "Index of..." but we're not there yet.